CefSharp.BrowserSubprocess.Core/JavascriptMethodHandler.cpp

```diff
@@ -38,7 +38,7 @@ namespace CefSharp
         }
         catch (Exception^ ex)
         {
-            exception = StringUtils::ToNative(ex->Message);
+            exception = StringUtils::ToNative(ExceptionExtension::CreateDetailedExceptionString(ex, ExceptionExtension::INNER_EXCEPTION_LOG_COUNT));
```

Comment:
No it does not compile when removing the constant.
When adding [System::Runtime::InteropServices::DefaultParameterValue(INNER_EXCEPTION_LOG_COUNT)] to the parameter of CreateDetailedExceptionString it will compile but then the C# version does not work anymore as it is not possible to define both the annotation and the = default parameter.
Thought about method overloading but that seemed a bit excessive as it is not resolved at compile time.